deadlock/deadlock.tex

```diff
@@ -396,7 +396,7 @@ \subsection{Partial Ordering (Dijkstra's Solution)}
 Philosopher $1$ gets fork $1$, Philosopher $2$ gets fork $2$, and so on until we get to Philosopher $n$.
 They have to choose between fork $1$ and $n$.
 fork $1$ is already held up by philosopher $1$, so they can't pick up that fork, meaning he won't pick up fork $n$.
-We have broken circular wait! Meaning deadlock isn't possible.
+We have broken the condition of circular wait! Meaning deadlock isn't possible.
```

Comment:
This is a nomenclature piece in computer science, `the condition of` is unneeded.